docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# T<sub>2</sub> Hahn Characterization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The purpose of the $T_2$ Hahn Echo experiment is to determine $T_2$ qubit property. \n",
+    "\n",
+    "In this experiment, we would like to get a more precise estimate of the qubit's decay time. $T_2$ represents the amount of time required for the transverse magnetization to fall to approximately 37% ($\\frac{1}{e}$) of its initial value.\n",
+    "\n",
+    "Since the qubit exposed to other noises (like $T_1$), we are using a $Rx(\\pi)$ pulse for decoupling and to solve our inaccuracy for the qubit frequency estimation."
```

Comment:
```suggestion
    "Since the qubit is exposed to other types of noise (like $T_1$), we are using a $Rx(\\pi)$ pulse for decoupling and to solve our inaccuracy for the qubit frequency estimation."
```

Reply:
changed as suggested.